### PR TITLE
Add Freezing Option to Benchmarking

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #105616

cc @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @aakhundov

### CODE REVIEW

File: .github/workflows/inductor-perf-test-nightly.yml [hunk 1]
```diff
@@ -35,6 +35,11 @@ on:
         required: false
         type: boolean
         default: false
+      freezing_cudagraphs:
+        description: Run inductor_cudagraphs with freezing for inference?
+        required: false
+        type: boolean
+        default: false
```
Comment (eellison) [inline .github/workflows/inductor-perf-test-nightly.yml:42]: does this need to be on for it to run in the nightly @desertfire ?

Comment (desertfire) [reply .github/workflows/inductor-perf-test-nightly.yml:42]: Yes, I guess you don't need it to run nightly?

**Review by desertfire — APPROVED** [review]

File: .github/workflows/inductor-perf-test-nightly.yml (hunk 1 above)
Comment (eellison) [reply .github/workflows/inductor-perf-test-nightly.yml:42]: Nightly would be nice i guess. I'll test it out first non nightly and see that everything is working okay then go from there.

**Pull request closed without merging**